tests/structured_conf/test_structured_config.py

```diff
@@ -1010,6 +1010,26 @@ def test_str2user(self, module: Any) -> None:
             # bad key
             cfg[Color.BLUE] = "nope"
 
+    def test_str2user_with_field(self, module: Any) -> None:
```

Comment:
Is this testing anything not already covered?

Reply:
No.
I'll remove it.

Reply:
I am trying to focus on specific aspects in tests, not on specific test classes (that might have a bunch of different aspects in them).
Let's remove it unless you think it's covering something not yet covered (and in that case, we should probably be more pointed in the test).

Reply:
Removed in https://github.com/omry/omegaconf/pull/502/commits/1019df6dac36e827789a6a3040557c10f5d3cf1d.